test/command_line/test_dose_analysis.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for dials.dose_analysis"""
+import procrunner
+
+
+def test_dose_analysis_dials_data(dials_data, tmpdir):
+    """Test dials.dose_analysis on scaled data."""
+    location = dials_data("l_cysteine_4_sweeps_scaled")
+    refls = location.join("scaled_20_25.refl").strpath
+    expts = location.join("scaled_20_25.expt").strpath
+
+    command = ["dials.dose_analysis", refls, expts, "min_completeness=0.4", "-v"]
+    result = procrunner.run(command, working_directory=tmpdir)
```

Comment:
Can the tests not call the `dials.command_line.dose_analysis.run()` function directly rather than via `procrunner`?

Reply:
I prefer to use procrunner so that we're testing the program as a user would on the command line, and to avoid potential errors in setting up the correct input for the tests.

Reply:
As discussed elsewhere, calling `dials.command_line.dose_analysis.run()` directly would still exercise all of the command line parsing code.

Reply:
I'm convinced on this one now, anecdotally test time went from about 17s to about 6s.